ci/prow/config_knative.yaml

```diff
@@ -166,6 +166,9 @@ periodics:
     - custom-job: istio-1.2-no-mesh
       full-command: "./test/e2e-tests.sh --istio-version 1.2-latest --no-mesh"
       dot-dev: true
+    - custom-job: gloo-0.17.1
```

Comment:
you may want to remove patch version just like `istio`, so there is no need to update everywhere with patch releases

Reply:
this would require another PR for `serving`. if you don't mind, i'll leave it on `0.17.1` for now and address this in a follow up?

Reply:
sgtm